README.md

```diff
@@ -21,10 +21,37 @@ To run, `pip install vinyldns-python` and then:
 >>> local_client.list_zones()
 ```
 
+## Contributing
+
+**Requirements**
+
+* `python3`
+* `pip`
+* `virtualenv`
+
+To get started, you will want to setup your virtual environment.  Ensure that you have `virtualenv`
+installed `>pip install virtualenv`.  Then, simply run `./bootstrap.sh` to setup your environment (unless
+you really want all these dependencies to be installed locally, which we do not recommend).
+
+Activate your virtual environment `>.virtualenv/bin/activate` and you will be ready to go!
```

Comment:
Should this be `source .virtualenv/bin/activate`? Running it directly didn't seem to throw me into the `(.virtualenv)` prompt and also required adding permissions (`chmod +x`) after installing virtualenv.